src/pages/index.tsx

```diff
@@ -22,13 +22,15 @@ import {
 
 const Home = () => {
   useEffect(() => {
-    Aos.init({
-      delay: 50,
-      offset: 200,
-      once: false,
-      mirror: true,
-      duration: 1000,
-    });
+    if (typeof window !== 'undefined') {
```

Comment:
I did a test simulating `user-agent` as `googlebot` and the problem persists :/

Let's try this here in this `useEffect` ? https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgent#examples

```js
if(useAgent !== {GOOGLEBOT}) // init Aos
```

Reply:
I will try here